src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,28 @@
+package calculator;
+
+import calculator.calc.Calculator;
+import calculator.parse.ExpressionParser;
+
+public class StringCalculator {
+
+    private ExpressionParser parser = new ExpressionParser();
+    private Calculator calculator = new Calculator();
+
+    public StringCalculator(String originExpression){
+        this.parser.parse(originExpression);
+    }
+
+    public int calculate(){
```

Comment:
```suggestion
    public int calculate() {
```

이러한 띄어쓰기들을 잘 지켜보면 좋을거 같습니다.

IntelliJ에서는 reformat code 기능이 존재하는데 한번 사용해보시면 좋을거 같아요~